lib/src/sip_ua_helper.dart

```diff
@@ -103,6 +103,7 @@ class SIPUAHelper extends EventManager {
     _settings.display_name = uaSettings.displayName;
     _settings.authorization_user = uaSettings.authorizationUser;
     _settings.user_agent = uaSettings.userAgent ?? DartSIP_C.USER_AGENT;
+    _settings.register_expires = uaSettings.register_expires;
```

Comment:
Forgot to copy the `register` parameter here?
```dart
    _settings.register = uaSettings.register;
```

Reply:
Oops. I'll do a new clean one?

Reply:
I think just add it, and it can be merged if the test is correct.

Reply:
Done.